lib/plugins/plugin-validator.spec.js

```diff
@@ -40,6 +49,8 @@ describe('plugin-validator', () => {
 
   afterEach(() => {
     jest.clearAllMocks()
+    // Clear Polyfill process.exitCode property for Node.js < v20
+    process.exitCode = undefined
```

Comment:
I think we should be resetting `exitCode` to its original value rather than always setting it to undefined.

Reply:
Sorry @nataliecarey this is my fault

I suggested [**default:** `undefined` from Node.js 20](https://nodejs.org/docs/latest-v20.x/api/process.html#processexitcode_1)

The default isn't documented in earlier versions but it's still `undefined` due to lack of assignment

Reply:
Resetting to original exitCode as requested

Reply:
Can you double check that `Object.defineProperty()` in Node.js 20 doesn't update the existing property and by reference override `originalExitCode` too?

Hopefully not, but may mean that Node.js 20 gets restored to the mock version again

Might need [`structuredClone()`](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) to detach the reference to the original object

Reply:
Anyone else think this is getting a little over complicated when the original fix was probably enough?  The tests are passing.

Reply:
Yeah definitely

Have to be aware that changing an object in JavaScript updates all the places it's referenced

Reply:
I've used the debugger in WebStorm and the previous value is undefined

Reply:
Ah brilliant, thank for checking